scripts/publish.sh

```diff
@@ -109,7 +109,12 @@ echo "Pushing to GitHub..."
 git push origin master --tags
 echo "Pushed to GitHub."
 
+echo "Publishing Docker image..."
+
```

Comment:
Missing a step here?

Reply:
Derp - should have cleaned this up, was a remnant of a different approach